src/styles/blocks/catalog.scss

```diff
@@ -18,8 +20,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: transform 300ms, color 300ms;
```

Comment:
would be a nice idea to create a variable for transition-duration as well, as usually you use same transition-duration across all your project